src/app/app.component.html

```diff
@@ -3,4 +3,51 @@ <h1>
     Najlepsza {{ title }} w sieci!
   </h1>
   <img width="300" alt="Angular Logo" src="http://vitaltrading.pl/wp-content/uploads/2016/04/checklist-1000x675.jpg">
-</div>
\ No newline at end of file
+</div>
+
+
+<input [(ngModel)]="newTask" placeholder="Wpisz zadanie do wykonania" (keyup.enter)='pushTask()' class="addTask">
+<button (click)="pushTask()">Dodaj Zadanie</button>
+<button type='sort' (click)=sortAZ()>Alfabetycznie</button>
+<button type='sort' (click)=sortZA()>Sort od końca</button>
+<button type='sort' (click)=sortNowe()>Najnowsze</button>
+<button type='sort' (click)=sortStare()>Najstarsze</button>
+
+
+<div class="checkbox">
+  <ul>
+    <li *ngFor="let task of tasks |filter:searchText; let ind = index ">
+      <label ng-repeat="task in tasks">
```

Comment:
Proszę usuń to ;). `ng-repeat` jest z AngularJS. Ogólnie nie wiem czy to jak to wygląda ma związek z zastosowaną biblioteką którą dociągnąłeś ale jak już napiszesz ją samodzielnie to postaraj się to też przebudować.

Reply:
https://github.com/jedi800/Checklistaa/pull/1/commits/66add8f495af27f61436b5c81d1539a320b0fd83